src/pages/docs/infrastructure/deployment-targets/tentacle/tentacle-communication.md

```diff
@@ -18,7 +18,7 @@ In listening mode Tentacle is the TCP server, and Octopus is the TCP client.
 :::figure
 ![Octopus to Listening Tentacle communication](/docs/infrastructure/deployment-targets/tentacle/images/listening-tentacle.png)
 :::
-
+ 
```

Comment:
Drive-by comment: remove this whitespace please!

Reply:
Was testing and forgot to revert, cheers!